### DISCUSSION TITLE
1.6.7 version question

### DISCUSSION THREAD

**Discussion by konstjar** [OHDSI/Achilles#572] [General]
Is there a plan to make v1.6.7 release in GitHub or at least a tag? There are differences between documentation and actual GitHub state:

Documentation:
![image](https://user-images.githubusercontent.com/4144173/114706119-e0297c80-9d28-11eb-9c4d-deaa75850a5f.png)

Actual GitHub state:
![image](https://user-images.githubusercontent.com/4144173/114706214-fafbf100-9d28-11eb-81ac-5515b0e7335f.png)

**Comment by fdefalco** [comment]
Yes, we're planning on making a new release in the coming weeks.  We're trying to get to a stable place before doing so and there are still some issues trickling in from the community in Europe that are making extensive use of the package.

**Reply by konstjar** [reply]
Great! Thank you for sharing this. 

Also I think it should be mentioned that v1.6.7 supports OMOP CDM v5.3.x only. I had few issues running it on v5.2 datasets.

**Comment by konstjar** [comment]
@fdefalco Do you have any updates on plans :-) ?

**Reply by fdefalco** [reply]
Yes. :) Will be releasing near the symposium.
